.github/workflows/lint.yml

```
@@ -0,0 +1,18 @@
+name: Lint
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+permissions:
+  contents: read
+jobs:
+    style:
+      name: PHP Style Check
+      uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@add-rules-for-cs-fixer
```

change to `main` once https://github.com/GoogleCloudPlatform/php-tools/pull/152 is merged
```suggestion
      uses: GoogleCloudPlatform/php-tools/.github/workflows/code-standards.yml@main
```